clients/gen/python.sh

```diff
@@ -26,7 +26,7 @@ readonly CLEANUP_DIRS
 # shellcheck source=./clients/gen/common.sh
 source "${CLIENTS_GEN_DIR}/common.sh"
 
-VERSION=2.5.1
+VERSION=2.6.0
```

Comment:
Does this need its own PR? Also, does that mean we need to regenerate the python client? That should have been set when we were releasing the 2.6.0 client.

Reply:
I’m not at home, but I believe that what is missing from the client [release process](https://github.com/apache/airflow-client-python/blob/main/dev/README_RELEASE_CLIENT.md) is “Update python.sh in airflow/main with the new client version”. Changing the version for generating the client is part of the process, (see above) we can inspect the generated client, the API version is specified here for instance https://github.com/apache/airflow-client-python/blob/main/airflow_client/README.md#unknown which is correct.

PR to update the client release process is welcome. I think we have the same issue with the go client.

Reply:
Yeah, the version seems to be correct. I think I will move the change to a separate PR and document the missing piece in the release process docs.